docusaurus.config.js

```diff
@@ -108,11 +112,7 @@ const config = {
             items: [
               {
                 label: 'Discord',
-                href: 'https://discord.com/channels/905194001349627914/941708033434738768',
-              },
-              {
-                label: 'Hyperledger Aries Mailing List',
-                href: 'https://lists.hyperledger.org/g/aries',
+                href: 'https://discord.com/channels/1022962884864643214/1179453305856991263',
               },
               {
                 label: 'Working Group Call Notes',
```

Comment:
Should point to wiki in github

Reply:
Also working group calendar can be removed, we have the wiki that lists both (line below)